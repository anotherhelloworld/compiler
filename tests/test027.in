var
  x: 1..10;
  y: 'a'..'z';
  z: pear..orange;