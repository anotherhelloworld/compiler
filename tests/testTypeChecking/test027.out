begin
         a
      []
         0
   :=
      1
end
